docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -0,0 +1,150 @@
+********************************
+Configuring Grafana and InfluxDB
+********************************
+
+This procedure describes how to display Sawtooth network statistics and operating system metrics in Grafana, using Docker, InfluxDB, and Telegraf.
+
+Requirements
+============
+
+This guide assumes Docker and one or more sawtooth validators have been configured.
+
+Download sawtooth-core from the Hyperledger GitHub repository.
+
+.. code-block:: console
+
+   $ git clone https://github.com/hyperledger/sawtooth-core.git
+
+InfluxDB
+========
+
+#. First, pull the InfluxDB Docker container from Docker Hub.
+
+   .. code-block:: console
+
+     $ docker pull influxdb
+
+#. Next, start the InfluxDB Docker container with persistent storage on the local file system and port 8086 exposed to the network. We also configure the database, admin, and regular users here.
+
+   .. note::
+
+      Depending on the host operating system /var/lib/influx-data may not be appropriate, modify this path accordingly.
+
+   .. note::
+
+      Be sure to replace these passwords with something more secure! Some characters such as ``,@!$`` have special meanings in the shell and may cause unexpected results if not properly escaped in the following command.
+
+   .. code-block:: console
+
+     $ sudo mkdir /var/lib/influx-data
+     $ docker run -d -p 8086:8086 -v /var/lib/influx-data:/var/lib/influxdb \
+       -e INFLUXDB_DB=metrics -e INFLUXDB_HTTP_AUTH_ENABLED=true \
+       -e INFLUXDB_ADMIN_USER=admin -e INFLUXDB_ADMIN_PASSWORD='admin' \
+       -e INFLUXDB_USER=lrdata -e INFLUXDB_USER_PASSWORD='test' \
+       --name sawtooth-stats-influxdb influxdb
```

Comment:
imo, the notes should come after the command. I believe after the command is consistent with the rest of the docs.

Reply:
Not necessarily - it depends on the note.  But I shall reorganize this step to make the nodes clearer.

Reply:
I moved the notes below the command as suggested.